src/metabase/notification/payload/impl/card.clj

```diff
@@ -43,6 +47,9 @@
   (let [{:keys [notification_card card_part]} payload
         send-condition                        (:send_condition notification_card)]
     (cond
+      (true? (-> notification_card :card :archived))
```

Comment:
nit: I see why you've "split" things, but would prefer threading to be everything or nothing. Don't like doing acrobatics with my eyes!